functions/periodicMetrics.js

```diff
@@ -17,28 +24,27 @@ module.exports.handler = async function(event, context) {
     parseInt(METRICS_PING_PERIOD, 10) || DEFAULT_METRICS_PING_PERIOD;
 
   let pingCount = 0;
-  logInfo("Periodic metrics monitor start");
+  log.debug("start");
   while (context.getRemainingTimeInMillis() > pingPeriod + 1000) {
+    pingCount++;
     try {
-      await sendHeartbeatMetrics(process.env, context);
+      await sendHeartbeatMetrics(log, process.env, context);
       pingCount++;
```

Comment:
There's a new line, 29, that's also incrementing `pingCount`.  Was this line meant to be removed?

Reply:
Removed line 29 - count increment here in the try/catch is probably the better place